runtime/gc_base/modronapi.hpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 1991, 2021 IBM Corp. and others
+ * Copyright (c) 1991, 2022 IBM Corp. and others
```

Comment:
There is no changes in this file overall. Copyright dates should not be changed and file should be excluded from change set.